cmd/root.go

```diff
@@ -66,6 +67,7 @@ func init() {
 	RootCmd.PersistentFlags().StringVar(&variabledataservice, "dataservice", "", "Optional, the name of a private data service in which the variable should be created. Leave blank to store in the default data service.")
 	RootCmd.PersistentFlags().BoolVarP(&verbose, "verbose", "v", false, "Display all possible logging info")
 	RootCmd.PersistentFlags().StringVarP(&appName, "app", "a", "", "Filter retrieve/deploy to a specific Skuid App")
+	RootCmd.PersistentFlags().BoolVarP(&nozip, "nozip", "z", false, "Ask for site not to be zipped")
```

Comment:
I'm curious why you added you this - is it necessary to have this to be able to pull by app name or did it seem like a good feature to add?

Reply:
It's something that pliny supports that skuid-cli... did not.

Reply:
Is this something that `deploy` should support as well?

Reply:
Not necessarily. At least not for now.